applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -154,7 +154,149 @@ def test_unsteady_thermal_heat_flux_3D10N(self):
         self.assertAlmostEqual(1.2294110493528096, temp)
+ 
+    def test_thermal_fixed_temperature_2D3N(self):
+        test_name = 'test_thermal_fixed_temperature_2D3N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[37]
+        self.assertAlmostEqual(13.08528783780587, temp)
+
+    def test_thermal_fixed_temperature_2D6N(self):
+        test_name = 'test_thermal_fixed_temperature_2D6N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[57]
+        self.assertAlmostEqual(13.08528783780587, temp)
+
+    def test_thermal_fixed_temperature_2D10N(self):
+        test_name = 'test_thermal_fixed_temperature_2D10N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[77]
+        self.assertAlmostEqual(13.08528783780587, temp)
+
+    def test_thermal_fixed_temperature_2D15N(self):
+        test_name = 'test_thermal_fixed_temperature_2D15N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[97]
+        self.assertAlmostEqual(13.08528783780587, temp)
+
+    def test_thermal_fixed_temperature_2D4N(self):
+        test_name = 'test_thermal_fixed_temperature_2D4N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[18]
+        self.assertAlmostEqual(4.9497705225, temp)
+
+    def test_thermal_fixed_temperature_2D8N(self):
+        test_name = 'test_thermal_fixed_temperature_2D8N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[50]
+        self.assertAlmostEqual(4.9497705225, temp)
+
+    def test_thermal_fixed_temperature_2D9N(self):
+        test_name = 'test_thermal_fixed_temperature_2D9N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[63]
+        self.assertAlmostEqual(4.9497705225, temp)
         
```

Comment:
This  delimiting line has a few white spaces. I do not know why Codacy does not report it.

Reply:
Removed.